doc/01-About.md

```diff
@@ -3,13 +3,13 @@
 Icinga DB is a set of components for publishing, synchronizing and
 visualizing monitoring data in the Icinga ecosystem, consisting of:
 
-* The Icinga DB daemon, which synchronizes monitoring data between a Redis server and a database
+* The Icinga DB daemon, which synchronizes monitoring data between a Redis®[*](TRADEMARKS.md#redis) server and a database
```

Comment:
Same as in README.

Reply:
Also same as in README: https://github.com/Icinga/icingadb/pull/736/files#r1560593740